test/include/util/performance_counter_test_util.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include "common/performance_counter.h"
+
+namespace terrier {
```

Comment:
https://github.com/cmu-db/terrier/wiki/Coding-Guideline

We basically set a rule that only a directory gets its own test util.